agent/src/main.cpp

```diff
@@ -10,12 +29,13 @@ void usage(const char *argv0)
 {
     printf("SoftwareContainer agent, v.%s\n", PACKAGE_VERSION);
     printf("Usage: %s [-p or --preload <num>] [-u or --user <uid>]", argv0);
-    printf("[-s or --shutdown <bool>] [-t or --timeout <seconds>]\n");
+    printf("[-s or --shutdown <bool>] [-t or --timeout <seconds>] [-c or --configPath <path>]\n");
     printf("\n");
     printf("--preload <num>     : Number of containers to preload, defaults to 0\n");
     printf("--user <uid>        : Default user id to be used when starting processes in the container, defaults to 0\n");
     printf("--shutdown <bool>   : If false, containers will not be shutdown on exit. Useful for debugging. Defaults to true\n");
     printf("--timeout <seconds> : Timeout in seconds to wait for containers to shutdown, defaults to 2\n");
+    printf("--configPath <path> : Path to a file or directory where Service Manifests exist, defaults to \"\"");
```

Comment:
We might very well add a "SoftwareContainer configuration" concept later, so I think we should use an option name that is more specific for this. Like, service manifest path or something like that.